src/main/java/de/tum/in/www1/artemis/service/ExamService.java

```diff
@@ -19,10 +19,7 @@
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestBody;
 
-import de.tum.in.www1.artemis.domain.Exercise;
-import de.tum.in.www1.artemis.domain.ProgrammingExercise;
-import de.tum.in.www1.artemis.domain.Result;
-import de.tum.in.www1.artemis.domain.User;
+import de.tum.in.www1.artemis.domain.*;
```

Comment:
Codacy found an issue: [Avoid unused imports such as 'de.tum.in.www1.artemis.domain'](https://app.codacy.com/gh/ls1intum/Artemis/pullRequest?prid=5968855)

Reply:
See f9e39e2dbe10ffa13483affe2de6359ac2244213